app/views/groups/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<nav>
+  <ul>
+    <li><%= link_to '<', user_groups_path(@user), :class=>'navBtn' %></li>
+    <li><p>CATEGORY</p></li>
+    <li><%= button_to '[>', destroy_user_session_path, :class=>'logout', method: :delete %></li>
+  </ul>
+</nav>
+<section class='addCategory'>
+  <h2>Add a New Category</h2>
+  <%= form_with model: @group, url: user_groups_path(@user), :class=>'forms', method: :post do |form| %>
+    <%= form.label :name, 'Name:' %>
+    <%= form.text_field :name, :class => 'texts' %>
+    <%= form.label :icon, 'Icon:' %>
+    <%= form.text_field :icon, :class => 'texts' %>
+    <div class='btnDiv'>
+      <%= form.submit 'Create Category', :class => 'btn' %>
+    </div>
+  <% end %>
+</section>
```

Comment:
- [ ] As stated in the project requirement when creating  a new category, indicating their name and icon (**Kindly make sure that both mandatory)**.
- When I try to omit the icon and submit it create without the icon as show below with the screenshot. 
![image](https://github.com/obibaadoma/Expense-App/assets/93598189/1dbfe328-f054-4819-a32e-a508e576dc2c)
![image](https://github.com/obibaadoma/Expense-App/assets/93598189/f4a62d44-10c9-4787-a3cf-d8c25a0267a9)

- When I try to omit the Name and submit it throw an error as show below with the screenshot. But Instead of throw an error I recommend you to create a validation that prohibited without the name and icon .
![image](https://github.com/obibaadoma/Expense-App/assets/93598189/6b6d24b4-85e4-47a2-8331-697ec5971a33)
![image](https://github.com/obibaadoma/Expense-App/assets/93598189/8d0aec51-0456-47ee-b76f-2ebdf98de889)

Reply:
thanks for the suggested i have worked on it accordingly